src/handler/mod.rs

```diff
@@ -187,12 +189,10 @@ pub struct Handler {
     enr: Arc<RwLock<Enr>>,
     /// The key to sign the ENR and set up encrypted communication with peers.
     key: Arc<RwLock<CombinedKey>>,
-    /// Pending raw requests.
+    /// Active requests that are awaiting a response.
     active_requests: ActiveRequests,
     /// The expected responses by SocketAddr which allows packets to pass the underlying filter.
     filter_expected_responses: Arc<RwLock<HashMap<SocketAddr, usize>>>,
-    /// Requests awaiting a handshake completion.
-    pending_requests: HashMap<NodeAddress, Vec<PendingRequest>>,
```

Comment:
Sending concurrent requests should still be done after the handshake, why was this removed?

Reply:
Ahh, yup makes sense. I re-introduced the `pending_requests` functionality